src/dnsimple/Services/Certificates.cs

```diff
@@ -213,7 +213,11 @@ public struct Certificate
         public string AuthorityIdentifier { get; set; }
         public DateTime CreatedAt { get; set; }
         public DateTime UpdatedAt { get; set; }
-        public DateTime? ExpiresOn { get; set; }
+        public DateTime? ExpiresAt { get; set; }
+
+        [Obsolete("ExpiresOn is deprecated, please use ExpiresAt instead.")]
+        [JsonIgnore]
+        public string ExpiresOn => ExpiresAt?.ToUniversalTime().ToString("yyyy-MM-dd");
```

Comment:
Should we keep the `DateTime?` data type? 

```suggestion
        public DateTime? ExpiresOn => ExpiresAt?.Date;
```

Reply:
IDK. for me it looks weird to have a DateTime only to have the date. The time usually will be 00:00:00. Wdyt?

Reply:
Like suggested by Simone, we dropped the ExpiresOn